src/index.js

```diff
@@ -50,9 +50,26 @@ const onfidoRender = (options, el, merge) => {
   return render( <Container options={options}/>, el, merge)
 }
 
+const base64Data = (dataURI) =>
+  ({
+    byteString: atob(dataURI.split(',')[1]),
+    mimeString: dataURI.split(',')[0].split(':')[1].split(';')[0]
+  })
+
+const getBlobfrom = ({byteString, mimeString}) => {
+  const buffer = new ArrayBuffer(byteString.length)
+  const integersArray = new Uint8Array(buffer)
+  for (let i = 0; i < byteString.length; i++) {
+    integersArray[i] = byteString.charCodeAt(i)
+  }
+  return new Blob([buffer], {type: mimeString})
+}
+
 
-const stripOneCapture = ({image,documentType,id}) =>
-  documentType === undefined ? {id,image} : {id,image,documentType}
+const stripOneCapture = ({image: img ,documentType,id}) => {
```

Comment:
Yes, it was named `blob` in my previous commit. I decided to change it to `image` to have a lower impact on clients who have already integrated but I agree with you that name should be more meaningful.